### DISCUSSION TITLE
part-ii/2021-54/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#167] [Question comments]
Part II 2021 Dynamical Systems: Paper 4, Section II, A 

https://questions.tripos.org/part-ii/2021-54/

**Comment by eha1234** [comment]
<details><summary>(ii)</summary>

1, 1, 2 respectively
</details>

<details><summary>(iii)</summary>

Only need to find the first few intersections, not all of them, to generate the rest of the cycle. You can do this by sketching the first half of the graph for $F^2, F^4, F^5$ and calculating intersections with diagonal. Got $1/3, 5/6$ for the two cycle, $2/9, 13/18, 5/9, 8/9$ for the four cycle, and $1/9, 11/18, 7/9, 4/9, 17/18$, $2/15, 19/30, 11/15, 8/15, 14/15$ for the two 5 cycles. 
</details>

Comments and corrections appreciated!
